auth-api/src/auth_api/services/api_gateway.py

```diff
@@ -213,6 +215,36 @@ def _consumer_exists(cls, email, env):
     @classmethod
     @user_context
     def _create_payment_account(cls, org: OrgModel, **kwargs):
+        """Create a payment account for environment.
+
+        Steps:
+        - Get the payment account details.
+        - Mask the details and call pay endpoint to create a account.
+        """
+        if not current_app.config.get("PAY_API_URL"):
```

Comment:
Which PAY_API_URL is this? will it ever be undefined? if it is, shouldn't we exception?